anago

```diff
@@ -230,76 +212,6 @@ copy_logs_to_workdir () {
   logrun -s cp -f $logfiles $WORKDIR
 }
 
-###############################################################################
-# Ensures we have write access to a specified registry
-# @param registry - A registry to check the ACLs for
-#
-ensure_registry_acls () {
```

Comment:
This function has one commented reference: https://github.com/kubernetes/release/blob/313f232977d63876d856647a0f9fd10aeb11bd59/anago#L260

Should we also remove that reference as we're removing the function?

Reply:
Yep, removed that line as well as the comment above :+1: